controllers/configurationpolicy_controller.go

```diff
@@ -1011,6 +1011,22 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 
 				// Otherwise, set the resolved data for use in further processing
 				plc.Spec.ObjectTemplates[i].ObjectDefinition.Raw = resolvedTemplate.ResolvedJSON
+			} else if isRawObjTemplate {
+				// no templates to resolve, but the template resolver will handle YAML conversion
+				resolvedTemplate, tplErr := tmplResolver.ResolveTemplate(rawData, nil)
+				if tplErr != nil {
+					addTemplateErrorViolation("Error parsing raw template", err.Error())
+				}
+				err := json.Unmarshal(resolvedTemplate.ResolvedJSON, &objTemps)
```

Comment:
It also seems that `objTemps` shouldn't be defined outside of the for loop since it's not used outside of it.